README.md

```diff
@@ -1219,6 +1219,14 @@ If you are using Linux on the desktop, you may want to use `/usr/bin/pinentry-gn
     $ pkill ssh-agent ; pkill gpg-agent ; \
       eval $(gpg-agent --daemon --enable-ssh-support --use-standard-socket \
       --log-file ~/.gnupg/gpg-agent.log --write-env-file)
+      
+Zsh note - if you're using zsh, add gpg-agent to your .zshrc plugins:
```

Comment:
Yup, that would be correct. If you want me to change that to reflect that it's oh-my-zsh specific, let me know. If that's too narrow a focus for this document, no worries.